regnety/dataset/imagenet.py

```diff
@@ -35,32 +36,25 @@ class ImageNet:
     If `augment_fn` argument is 'val', then the images will be center cropped to 224x224.
 
     Args:
-        tfrecs_filepath: list of filepaths of all TFRecords files
-        batch_size: batch_size for the Dataset
-        image_size: final image size of the images in the dataset
-        augment_fn: function to apply to dataset after loading raw TFrecords
-        num_classes: number of classes
-        
+       cfg: regnety.regnety.config.config.PreprocessingConfig instance.
     """
     def __init__(
         self,
-        tfrecs_filepath: List[str]  = None,
-        batch_size: int = 1024,
-        image_size: int = 512,
-        augment_fn: Union[str, Callable]  = "default",
-        num_classes: int = 1000,
-        scale_to_unit: bool = True
+        cfg
     ):
 
-        if (tfrecs_filepath is None) or  (tfrecs_filepath == []):
+        self.tfrecs_filepath = cfg.tfrecs_filepath
+        self.batch_size = cfg.batch_size
+        self.image_size = cfg.image_size
+        self.crop_size = cfg.crop_size
+        self.resize_pre_crop = cfg.resize_pre_crop
+        self.augment_fn = cfg.augment_fn
+        self.num_classes = cfg.num_classes
+        self.color_jitter = cfg.color_jitter
+        self.scale_method = cfg.scale_method
```

Comment:
Spaces between arguments and their values.

Reply:
Sorry, but these are variable assignments.